ci/test/00_setup_env_mac_host.sh

```diff
@@ -9,7 +9,8 @@ export LC_ALL=C.UTF-8
 export HOST=x86_64-apple-darwin18
 export PIP_PACKAGES="zmq"
 export GOAL="install"
-export BITCOIN_CONFIG="--with-gui --enable-reduce-exports --enable-werror --with-boost-process"
+# ELEMENTS: add -fno-stack-check to work around clang bug on macos
+export BITCOIN_CONFIG="--with-gui --enable-reduce-exports --enable-werror --with-boost-process CXXFLAGS=-fno-stack-check"
```

Comment:
hmm, this should be temporary, right?

Reply:
Yes, just until Cirrus updates their MacOS instances.